R/fullVALIDATIONREMIND.R

```diff
@@ -136,11 +136,28 @@ fullVALIDATIONREMIND <- function(rev = 0) {
   # CEDS Emissions ----
 
   # Historical emissions from CEDS data base
-  calcOutput(
-    "Emissions", datasource = "CEDS2024", file = valfile,
-    aggregate = columnsForAggregation, append = TRUE, warnNA = FALSE,
-    try = FALSE, writeArgs = list(scenario = "historical", model = "CEDS")
-  )
+  ceds <- calcOutput(
+    "Emissions", datasource = "CEDS2024",
+    aggregate = columnsForAggregation, warnNA = FALSE, try = FALSE)
+
+  # the following variables only have meaningful data on global level
+  # as bunkers exist in source only on the global level and were only added to
+  # regions because madrat doesn't allow global data before this point
+  # (see mrcommons::convertCEDS2024)
```

Comment:
we have encountered similar problems before, and there is a solution that can be applied to the calc function directly. see `calcIEA_ETP`